pvanalytics/features/orientation.py

```diff
@@ -0,0 +1,193 @@
+    # that takes a Series and returns the :math:`r^2` for a curve fit.
+    # If the two conditions are not met then `default` is returned and
+    # no curve fitting is performed.
+    high_enough = True
+    if peak_min is not None:
+        high_enough = day.max() > peak_min
+    if (_hours(day, freq) > min_hours) and high_enough:
+        return fitfunc(day)
+    return default
+
+
+def _freqstr_to_hours(freq):
+    # Convert pandas freqstr to hours (as a float)
+    if freq.isalpha():
+        freq = '1' + freq
+    return pd.to_timedelta(freq).seconds / 60
+
+
+def _hours(data, freq):
+    # Return the number of hours in `data` with timestamp
+    # spacing given by `freq`.
+    return data.count() * _freqstr_to_hours(freq)
+
+
+def _group_by_day(data):
+    # Group data by timezone localized date.
+    #
+    # We use this function (rather than `data.resample('D')`) because
+    # Resampler.apply() makes the data tz-naive when passed to the
+    # function being applied. This causes the curve fitting functions
+    # to fail when the minute-of-the-day is used as the x-coordinate
+    # since removing timezone information can cause data for a day to
+    # span two dates.
+    #
+    # The date needs to be localized in the same timezone as
+    # data.index so we can easily reindex the series with the original
+    # index at a later time.
+    return data.groupby(
+        pd.to_datetime(data.index.date).tz_localize(data.index.tz)
+    )
+
+
+def tracking_nrel(power_or_irradiance, daytime, correlation_min=0.94,
+                  fixed_max=0.96, min_hours=5, peak_min=None,
+                  midday=None):
+    """Flag days that match the profile of a single-axis tracking PV system.
+
+    For the values on a day to be marked True, they must satisfy the
+    following three conditions:
+
+    1. a restricted quartic [#]_ must fit the data with :math:`r^2`
+       greater than `correlation_min`
+    2. the :math:`r^2` for a quadratic fit must be less than
+       `fixed_max`
+    3. the :math:`r^2` for the restricted quartic fit must be greater
+       than the :math:`r^2` for a quadratic fit
+
+    Values on days where any one of these conditions is not met are
+    marked False.
+
+    .. [#] The specific quartic used for this fit is centered within
+       70 minutes of 12:00, the y-value at the center must be within
+       15% of the median for the day, and it must open downwards.
+
+    Parameters
+    ----------
+    power_or_irradiance : Series
+        Timezone localized series of power or irradiance measurements.
+    daytime : Series
+        Boolean series with True for times that are during the
+        day. For best results this mask should exclude early morning
+        and late afternoon as well as night. These times may have
+        problems with shadows that interfere with curve fitting, but
+        do not necessarily indicate that the tracker is not working.
+    correlation_min : float, default 0.94
```

Comment:
```suggestion
    r2_min : float, default 0.94
```

R2 is the coefficient of determination (R is the correlation coefficient).